packages/design-system/package.json

```diff
@@ -44,7 +44,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/vue-fontawesome": "^3.0.3",
-    "element-plus": "^2.3.6",
+    "element-plus": "2.4.3",
```

Comment:
does it make sense to move this upgrade into a separate PR?

Reply:
Not sure here, the only changes left that are unrelated to this upgrade are few lines of css in [this file](https://github.com/n8n-io/n8n/pull/10281/files#diff-f9e39ebedc3435e839605b9e7c432fc08bb554caa5dc7bcd5751c9007d762282), so I would keep it together if there are no major objections.